src/apps/material/material.test.ts

```diff
@@ -0,0 +1,78 @@
+describe("Material app e2e", () => {
+  it("Should render the material page as expected", () => {
+    cy.log("Does the Material have title?");
+    cy.get(".text-header-h1").should("have.length.gte", 1);
```

Comment:
In general please check that the expected data is visible/accessible somewhere on the page instead of checking for the presence of specific HTML tags or classes.

This is recommended as best practice by Cypress to avoid frail tests.

This applies throughout the test.